### DISCUSSION TITLE
Font-Awesome or relative path for custom icons

### DISCUSSION THREAD

**Discussion by joadwe** [luizdepra/hugo-coder#729] [Ideas]
I wondered if it was possible to add support for the use of font-awesome or relative paths for to .svg icons on the home page?

**Comment by luizdepra** [comment]
Sure, it is possible. You could add the font-awesome includes into your project and use their CSS classes in the configuration.
But we are using fork-awesome icons. Don't they have the icons you want?

**Reply by joadwe** [reply]
Thanks for the quick reply! Fork-awesome is great, their selection just seems to be a little smaller. I'm a research scientist and some of the more fun/relevant scientific icons arent available https://fontawesome.com/search?q=vials&o=r. I am new to HUGO themes; i did try adding font awesome into the project but it appears to have the same naming structure as fork-awesome and the HUGO server would fail to load any icons if both were present. I will do some further reading to see where I'm going wrong.

**Reply by luizdepra** [reply]
If I remember correctly, you will need to remove [this line](https://github.com/luizdepra/hugo-coder/blob/5e4e500cdfe00babb29eb7aeef18c9d0ec3c5a47/assets/scss/coder.scss#L2) to disable fork-awesome. And, you will need to add the font awesome into the header section, [here](https://github.com/luizdepra/hugo-coder/blob/main/layouts/partials/head/extensions.html).

You could either fork the theme and change these files or simple override them by placing a file with the same name inside the same folder structure. Ex: to override `extension.html` you need to create the file in `your-project-path/layouts/partials/head/extensions.html`.

**Reply by joadwe** [reply]
Perfect, thanks for taking the time to share this!
